_opcodes/op74-bgee.html

```diff
@@ -0,0 +1,20 @@
+---
+n: 74
+name: "State: Blindness"
+param1: "Irrelevant"
+param2: "Irrelevant"
+bg1: 0
+bg2: 0
+bgee: 1
+iwd1: 0
+iwd2: 0
+pst: 0
+---
+Applies the blindness effect to the targeted creature(s). This sets <a href="../files/ids/bgee/stats.htm#147">VISUALRANGE</a> to 2, sets <a href="../files/ids/bgee/state.htm#BIT18">STATE_BLIND</a> to true and incurs a cumulative 4 point base THAC0 penalty plus a cumulative 4 point base AC penalty.<br />
```

Comment:
Can be merged back, just note the difference in the penalties (just rechecked that they are indeed -10 in vanilla bg2).

Reply:
Done.